yarn-project/pxe/src/contract_tree/index.ts

```diff
@@ -151,7 +151,7 @@ export class ContractTree {
   public async getContractMembershipWitness() {
```

Comment:
Can you expose the block number here?

It's a very useful functionality and we will need this for oracles.